submitter/__init__.py

```diff
@@ -14,3 +16,5 @@
 logger.info(
     "Logging configured with level=%s, filter=%s", CONFIG.LOG_LEVEL, CONFIG.LOG_FILTER
 )
+sentry_sdk.init(CONFIG.SENTRY_DSN, environment=CONFIG.ENV)
+logger.info("Sentry initialized with DSN=%s and env=%s", CONFIG.SENTRY_DSN, CONFIG.ENV)
```

Comment:
This feels a bit odd to log things like `INFO:submitter:Sentry initialized with DSN=None and env=dev`

Not a blocking comment, but maybe checking if `CONFIG.SENTRY_DSN` is set and only doing the `sentry_sdk.init` if it is set and logging something like `Not enabling sentry due to CONFIG.SENTRY_DSN not being set` would be clearer. Doing the init without a DSN seems like it could be problematic at some point even though it seems to be okay now.

Reply:
Yeah, the `sentry_sdk.init` `dsn` defaults to None so it doesn't hurt to initiate it with that but I agree the logging is confusing and there's not really a reason to initiate it without a DSN. Will update.